starter-code/index.js

```diff
@@ -1,17 +1,53 @@
 class SortedList {
-  constructor() {}
+  constructor(...items) {
+    this.items = items;
+    this.length = items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function(a, b){return a - b});
```

Comment:
use ES6 :) try to get familiar with the arrow function 

`    this.items.sort((a, b) => {return a - b});
`